.pipelines/templates/template-clean-e2e-db.yml

```diff
@@ -9,9 +9,11 @@ steps:
 
       export DATABASE_NAME=v4-e2e-$(git log --format=%h -n 1 HEAD)
 
-      run_rp
-      validate_rp_running
-      register_sub
-      run_e2e
-      kill_rp
-    displayName: "🚀 Run ARO RP & E2E"
+      if [ $RP_MODE = "development" ] 
+      then
+        clean_e2e_db
+      else
+        echo "### skipping db clean because not development rp"
+      fi
+    displayName: "🧹 Clean cluster DB"
+    condition: always()
```

Comment:
I think you should be able to just update the condition such as : 
`and(always(), eq(variables['RP_MODE'], 'development'))`